webservices/common/models/costs.py

```diff
@@ -0,0 +1,83 @@
+from webservices import utils
+from .base import db
+
+
+class CommunicationCost(db.Model):
+    __tablename__ = 'ofec_communication_cost_mv'
+
+    idx = db.Column(db.Integer, primary_key=True)
+    committee_id = db.Column('cmte_id', db.String, index=True)
+    candidate_id = db.Column('cand_id', db.String, index=True)
+    committee_name = db.Column('cmte_nm', db.String)
+    candidate_name = db.Column('cand_name', db.String)
+    candidate_office_state = db.Column('cand_office_st', db.String, index=True)
+    candidate_office_district = db.Column('cand_office_district', db.String, index=True)
+    candidate_office = db.Column('cand_office', db.String, index=True)
+    candidate_party_affiliation = db.Column('cand_pty_affiliation', db.String, index=True)
+    transaction_date = db.Column('_transaction_dt', db.Date, index=True)
+    transaction_amount = db.Column('transaction_amt', db.Numeric(30, 2), index=True)
+    purpose = db.Column(db.String)
+    communication_type = db.Column('communication_tp', db.String, index=True)
+    communication_class = db.Column('communication_class', db.String, index=True)
+    support_oppose_indicator = db.Column('support_oppose_ind', db.String, index=True)
+    image_number = db.Column('image_num', db.String, index=True)
+    line_number = db.Column('line_num', db.String, index=True)
+    form_type_code = db.Column('form_tp_cd', db.String, index=True)
+    schedule_type_code = db.Column('sched_tp_cd', db.String, index=True)
+    tran_id = db.Column(db.String)
+    sub_id = db.Column(db.Integer)
+    file_number = db.Column('file_num', db.Integer)
+    report_year = db.Column('rpt_yr', db.Integer, index=True)
+
+
+class Electioneering(db.Model):
+    __tablename__ = 'ofec_electioneering_mv'
+
+    idx = db.Column(db.Integer, primary_key=True)
+    form_type = db.Column('form_tp', db.String, index=True)
+    committee_id = db.Column('cmte_id', db.String, index=True)
+    candidate_id = db.Column('cand_id', db.String, index=True)
+    candidate_name = db.Column('cand_name', db.String, index=True)
```

Comment:
This column doesn't seem to be indexed in the materialized view, so we shouldn't mark it as indexed here. I don't think this would be especially useful to index, anyway, unless we want to expose a search on exact match on candidate names.